### ISSUE TITLE
Errors preventing startup: iron-router couldn't install npm packages

### ISSUE THREAD

**Issue by LynnRas** [iron-meteor/iron-router#467]
Running this simple app worked yesterday--I tried it with another app and had the same result: 

Lynns-MacBook-Pro:newkitsmart lynnrasmussen$ meteor
[[[[[ ~/Documents/code/newkitsmart ]]]]]

=> Meteor 0.7.1.2 is available. Update this project with 'meteor update'.
iron-router: updating npm dependencies -- connect...

npm WARN using --force I sure hope you know what you are doing.
npm WARN package.json packages-for-meteor-smartpackage-iron-router@0.0.0 No description
npm WARN package.json packages-for-meteor-smartpackage-iron-router@0.0.0 No repository field.
npm WARN package.json packages-for-meteor-smartpackage-iron-router@0.0.0 No README data
npm http GET https://registry.npmjs.org/connect/2.7.10
npm http GET https://registry.npmjs.org/connect/2.7.10
npm http GET https://registry.npmjs.org/connect/2.7.10
npm http GET https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm ERR! fetch failed https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm http GET https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm ERR! fetch failed https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm http GET https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm ERR! fetch failed https://registry.npmjs.org/connect/-/connect-2.7.10.tgz
npm ERR! Error: SELF_SIGNED_CERT_IN_CHAIN
npm ERR!     at SecurePair.<anonymous> (tls.js:1362:32)
npm ERR!     at SecurePair.EventEmitter.emit (events.js:92:17)
npm ERR!     at SecurePair.maybeInitFinished (tls.js:974:10)
npm ERR!     at CleartextStream.read [as _read](tls.js:462:15)
npm ERR!     at CleartextStream.Readable.read (_stream_readable.js:320:10)
npm ERR!     at EncryptedStream.write [as _write](tls.js:366:25)
npm ERR!     at doWrite (_stream_writable.js:221:10)
npm ERR!     at writeOrBuffer (_stream_writable.js:211:5)
npm ERR!     at EncryptedStream.Writable.write (_stream_writable.js:180:11)
npm ERR!     at write (_stream_readable.js:583:24)
npm ERR! If you need help, you may report this log at:
npm ERR!     http://github.com/isaacs/npm/issues
npm ERR! or email it to:
npm ERR!     npm-@googlegroups.com

npm ERR! System Darwin 11.4.2
npm ERR! command "/Users/lynnrasmussen/.meteor/tools/09b63f1ed5/bin/node" "/Users/lynnrasmussen/.meteor/tools/09b63f1ed5/bin/npm" "install" "--force"
npm ERR! cwd /Users/lynnrasmussen/.meteorite/packages/iron-router/EventedMind/iron-router/927bbaed1b87894fb6fb70807f74ff3a5bd89f29/.npm/package-new-ikiokd
npm ERR! node -v v0.10.22
npm ERR! npm -v 1.3.14
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/lynnrasmussen/.meteorite/packages/iron-router/EventedMind/iron-router/927bbaed1b87894fb6fb70807f74ff3a5bd89f29/.npm/package-new-ikiokd/npm-debug.log
npm ERR! not ok code 0

=> Errors prevented startup:

While building package `iron-router`:
error: couldn't install npm packages from npm-shrinkwrap

=> Your application has errors. Waiting for file change.

**Comment by diego2013** [comment]
@dejayc Thanks. That fixed the problem for me too and I was able to see that Iron-Router was successfully installed. Then I found that Iron router will only work with Meteor 0.9.0 and not the current version 1.0.3.1, but that's another story...

**Comment by cmather** [comment]
Hi @diego2013, iron:router works with the latest version of Meteor. Not sure why you're having an issue. Are you maybe using cmather:iron-router instead of iron:router?

**Comment by diego2013** [comment]
Hi @cmather , sorry for this late response (I marked this email as read but never took action on it). I thought I should reply and clarify about this issue. I think that was exactly what was happening. I wasn't careful enough when installing this package (probably didn't notice the difference back then when reading tutorials). I got it working the day after posting this. Currently using iron:router@1.0.7 w/ METEOR@1.1.0.2  and works like charm. Thank you for your time and help.
